### PR TITLE
fix(service-name-op): hide operator on `service_name`

### PR DESCRIPTION
**Pull request by svennergr**
It's not pretty, but it works...
Hiding the operator for `service_name`, as this one should always be an `equal` operator.

### CODE REVIEW

File: src/Components/IndexScene/LayoutScene.tsx [hunk 1]
```diff
@@ -117,6 +117,15 @@ function getStyles(theme: GrafanaTheme2) {
           },
         },
       },
+      // the `service_name` filter is a special case where we want to hide the operator
+      '[data-testid="AdHocFilter-service_name"]': {
```
Comment (matyax) [inline src/Components/IndexScene/LayoutScene.tsx:121]: 😬

**Comment by matyax** [comment]
I would get rid of this code which is not really doing anything: https://github.com/grafana/explore-logs/blob/main/src/Components/IndexScene/IndexScene.tsx#L164-L174


https://github.com/grafana/explore-logs/assets/1069378/f3e88fa0-aff9-4958-8bbb-c76ee48bb0a4

**Review by matyax — APPROVED** [review]

**Comment by svennergr** [comment]
> I would get rid of this code which is not really doing anything: https://github.com/grafana/explore-logs/blob/main/src/Components/IndexScene/IndexScene.tsx#L164-L174

You mean get rid of the condition?

**Comment by matyax** [comment]
Yeah, because only works when there's only 1 filter, otherwise it does nothing.

**Comment by svennergr** [comment]
> Yeah, because only works when there's only 1 filter, otherwise it does nothing.

This: https://github.com/grafana/explore-logs/pull/467/commits/2bbd528c15ab6f3f56a4d76c556df8efb0635f81 ?

**Review by matyax — APPROVED** [review]
:cohete:

**Pull request merged**